### PR TITLE
add isCoursePresent.middleware && little fixes

### PR DESCRIPTION
**Pull request by MikolaLvivUA**

### CODE REVIEW

File: src/routes/course/course.router.ts [hunk 1]
```diff
@@ -9,8 +10,8 @@ router.get('/:course_id', courseController.getCourseById);
 
 router.post('/', courseController.createCourse);
```
Comment (VictorKmin) [inline src/routes/course/course.router.ts]: router.use('/:course_id', isCoursePresent)
And romive it from another routes

File: src/routes/course/course.router.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { Router } from 'express';
 
 import { courseController } from '../../controllers';
+import { isCoursePresent } from '../../middleware/course';
```
Comment (VictorKmin) [inline src/routes/course/course.router.ts]: import { isCoursePresent } from '../../middleware';

File: src/routes/course/course.router.ts (hunk 1 above)
Comment (MikolaLvivUA) [reply src/routes/course/course.router.ts]: fixes

File: src/routes/course/course.router.ts (hunk 2 above)
Comment (MikolaLvivUA) [reply src/routes/course/course.router.ts]: fixed

**Pull request merged**